specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/stable/2016-05-01/Fabric.json

```diff
@@ -1,107 +1,214 @@
 {
- "swagger": "2.0",
```

Comment:
@deathly809 could you revert formatting so it is to review?

Reply:
I don't understand what you are asking.

Reply:
Sorry, I mean "it's easy to review". It's really hard to review if there are a lot of formatting changes, such as replacing indent from 4 symbols to 1 symbol. Please, have a look the file `Fabric.json` in https://github.com/Azure/azure-rest-api-specs/pull/2809/files

Reply:
Then some of my files with be 2 spaces and the others 4 spaces.  Would this be a temporary formatting change and then I could change it back?  I really don't want some files being 2 and some being 4.